corehq/apps/enterprise/enterprise.py

```diff
@@ -7,9 +7,11 @@
 
 from memoized import memoized
 
+from corehq.apps.reports.standard.deployments import _format_commcare_version
```

Comment:
It's a smell to import a private function. Should this now be a public function?

Reply:
c25a980ec516215fb65b406299624c21a66dbd46